bundles/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/layout/shape/Square.java

```diff
@@ -27,7 +28,7 @@
  * @author Jørgen Austvik - Initial contribution
  */
 @NonNullByDefault
-public class Square extends Shape {
+public class Square extends SingleLightShape {
```

Comment:
Do you mind adding comment why it is called a "Single"-LightShape instead of Shape?

Reply:
(Thinking out loud) Background: We have these things we can put on the wall. They have different shapes, hex, triangle, square, line. Most of the old style things have a single light. The Hexagon is either blue or red. They are described by one point, and as long as we have the outline of these elements, we can draw them with one algorithm. So this "SingleLightShape" is the thing that lets me write that algorithm once.

Then we have Elements Hegagons which have a light in each corner - so 6 colors and 6 panel ids in one thing you put on the wall. Images of lines also makes me believe lines can have two colors. 

So I was thinking "Shape" as something you put on the wall, and then the old style triangle/square/hex was those that had only one color. But what if we use the product names instead?

If I rename Shape to Panel, that would be the plastic thing that is placed on the wall.
Then I rename SingleLightShape to just Shape as that is the product name of those things with just one color.

Reply:
I never questioned that the name makes sense, I rather didn't know what the idea behind the name was. Ok, now I get it. Thanks.